packages/evolution-frontend/src/components/survey/Question.tsx

```diff
@@ -326,8 +327,9 @@ export class Question<CustomSurvey, CustomHousehold, CustomHome, CustomPerson> e
                 );
             }
         };
-        const content = (
+        const content = [(
             <div
+                key="content"
```

Comment:
keys are used to uniquely identify a react object and allow to only refresh the ones that change. They should be unique in the application, you can this `this.props.path` as suffix to the key, for example `content_${this.props.path}`